views/showcase.html

```diff
@@ -95,13 +95,17 @@ <h3 class="text-xl font-semibold">
             </dl>
           </a>
           {% endfor %}
+          {% for view in resource.views %}
```

Comment:
I'm not sure if views are property of a `resource` obj here - I think it's a property of `dataset` obj. Also, I think this should go together with the actual preview.

Reply:
`views` are placed in `resources` in `datapackage.json` file. If we relate `views` to `dataset` object, every `resource` will have `Preview`. May be I'm wrong here.
I also think that this should go with actual preview, but I found some inconsistencies here. Example page is:
https://www.opendata.dk/city-of-vejle/sygefravaer
and `datapackage.json file is:
https://www.opendata.dk/city-of-vejle/sygefravaer/datapackage.json
If you look at this file you can see that there is no `view` for second `resource`, but actually there is a preview for this resource. I can't figure out how this preview is generated without settings from the `datapackage.json` file.